hatohol.spec.in

```diff
@@ -253,9 +253,10 @@ rm -rf %{buildroot}
 %{_unitdir}/hatohol.service
 %{_tmpfilesdir}/hatohol.conf
 %endif
-%{_datadir}/hatohol/sql/create-db.sql
```

Comment:
Now 'create-db.sql' is being used only for test (TravisCI) from launch-hatohol-for-test.sh.
Why don't we stop installing it ?

I.e.  
The following lines are added in server/data/Makefile.am
<pre>
diff --git a/server/data/Makefile.am b/server/data/Makefile.am
index b22ebf6..c190228 100644
--- a/server/data/Makefile.am
+++ b/server/data/Makefile.am
@@ -1,5 +1,4 @@
 sql_DATA = \
-       create-db.sql \
        init-user.sql \
        server-type-zabbix.sql \
        server-type-nagios.sql \
@@ -14,6 +13,8 @@ sql_DATA = \
        incident-trackers.sql \
        actions.sql
 
+noinst_DATA = create-db.sql
+
 sqldir = $(pkgdatadir)/sql
 
 HATOHOL_SERVER_TYPE_UTIL = ../tools/hatohol-server-type-util
@@ -35,7 +36,8 @@ pkgsysconfdir = $(sysconfdir)/$(PACKAGE)
 
 EXTRA_DIST = \
        $(pkgsysconf_DATA) \
-       $(sql_DATA)
+       $(sql_DATA) \
+       $(noinst_DATA)
 
 CLEANFILES = \
        server-type-zabbix.sql \
</pre>
So we don't have to add %exclude in spec. file.